src/irclog2html/logs2html.py

```diff
@@ -211,6 +211,9 @@ def main(argv=sys.argv):
     parser.add_option('-g', '--glob-pattern', dest="pattern", default="*.log",
                       help="glob pattern that finds log files to be processed"
                       " (default: *.log)")
+    parser.add_option('-o', '--output-dir', dest="output_dir", default=None,
+                      help="destination output directory"
+                           " (default: same as input directory)")
```

Comment:
Okay, adding `default=None` did not help.

Ah, I see!  In `src/irclog2html/tests/test_logs2html.py` there are like three unit tests that create `optparse.Values()` objects directly!  These will need to have to be updated.

I don't know my own code, LOL.

Reply:
(On the plus side looking at my own stupid old code makes me realize how much I've improved as a programmer over the years ;)